src/pages/send-tokens/components/send-form-inner.tsx

```diff
@@ -73,9 +75,21 @@ export function SendFormInner(props: SendFormProps) {
   const onSubmit = useCallback(async () => {
     if (values.amount && values.recipient && values.txFee && selectedAsset) {
       selectedAsset.type === 'stx' && setAmount(values.amount);
+      if (values.txFee > 5) {
```

Comment:
[Magic number.](https://en.wikipedia.org/wiki/Magic_number_(programming)). The next person reading this may well question the relevance of a fee of this value. Why 5?

Let's move the value 5 somewhere else, like the constants file, named something descriptive like `HIGH_FEE_AMOUNT_STX`.

Reply:
Yep, good call, will change.